z3rro/src/test.rs

```
@@ -28,13 +28,14 @@ pub fn test_prove(f: impl for<'ctx> FnOnce(&'ctx Context, &mut SmtScope<'ctx>) -
 
     prover.add_provable(&theorem);
     match prover.check_proof() {
-        ProveResult::Counterexample => panic!(
+        Ok(ProveResult::Counterexample) => panic!(
             "counter-example: {:?}\nassertions:\n{:?}",
             prover.get_model(),
             prover.get_assertions()
         ),
-        ProveResult::Unknown(reason) => panic!("solver returned unknown ({})", reason),
-        ProveResult::Proof => {}
+        Ok(ProveResult::Unknown(reason)) => panic!("solver returned unknown ({})", reason),
+        Ok(ProveResult::Proof) => {},
+        Err(_) => {}
```

why is the error here silently ignored?